pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -287,7 +287,7 @@ func (c *Cluster) makeMonDaemonContainer(monConfig *monConfig) v1.Container {
 	}
 
 	// Add messenger 2 port
-	addContainerPort(container, "msgr2", 3300)
+	addContainerPort(container, "tcp-msgr2", 3300)
```

Comment:
There is another port up on line 266

Reply:
Or do we not need to update the names on the container ports? Is updating the service port sufficient?

Reply:
@leseb asked me to change the container port, but I dont think that istio checks this.

Reply:
should "client" be renamed to "tcp-msgr1"?

Reply:
> @leseb asked me to change the container port, but I dont think that istio checks this.

True, sounds good to keep the container port names consistent even if istio doesn't check them. 

> should "client" be renamed to "tcp-msgr1"?

Yes that sounds good

Reply:
will do.